protocol-definitions/custom/Custom.yaml

```diff
@@ -868,5 +868,74 @@ customTypes:
           type: Data
           nullable: true
           since: 2.0
-
-
+  - name: SqlQueryId
+    since: 2.1
+    params:
+      - name: memberIdHigh
+        type: long
```

Comment:
We prefer to use UUID type for the sake of being uniform. 
All other representations of UUID's directly use UUID type.

Reply:
`QueryId` is an internal object. It is constructed from several UUIDs, but it is not UUID itself. I think uniformity is not applicable here - the object is designed this way on the server-side for a reason.

Reply:
Resolving, since the query ID is a string now.